pup-0002.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Docstring Style Conventions
+author: Austin Macdonald
+created: 28-Mar-2017
+status: proposed
+---
+
+## Summary
+
+This PUP proposes the use of Google style docstrings for Pulp 3+.
+
+## Motivation
+
+Most current Pulp developers find Google Style docstrings to be more readable than the default
```

Comment:
I think we're motivated by two things:
* That the existing docstrings are not very readable
* That there is inconsistent style

This seems to focus on the solution versus a description of the problem with the current state.